### ISSUE TITLE
Absolute paths in webpack metadata for `ignored` modules.

### ISSUE THREAD

**Issue by adubovsky** [webpack/webpack#3154]
**I'm submitting a bug report**
**Webpack version:** 
2.x

**Please tell us about your environment:**
OSX 10.x

**Current behavior:**
Webpack metadata (`recordsPath` config option) contains absolute paths for `ignored` modules.

**Expected/desired behavior:**
No absolute paths in the metadata.

In our project we use `Intl` polyfill. When we import it into the project, the absolute paths appear in the webpack metadata:

In vendor.ts:

```
import 'intl';
import 'intl/locale-data/jsonp/en.js';
import 'intl/locale-data/jsonp/es.js';
```

Webpack records:

```
"node_modules/source-map-loader/index.js!node_modules/ts-helpers/index.js": 1054,
"ignored /Users/adubovsky/Projects/<project-name>/ionic-app/node_modules/intl ./locale-data/complete.js": 1055,
"node_modules/awesome-typescript-loader/dist/entry.js!app/app.module.ts": 1056,
```

Could you please help?

**Comment by jamestautges** [comment]
I think this is identical to #2991 which I've fixed in pull request #3147.

**Comment by adubovsky** [comment]
@jamestautges , thanks! it's the same issue, looking forward for your PR merge )
